narwhals/series.py

```diff
@@ -1810,9 +1961,53 @@ def head(self, n: int = 5) -> Series:
                "zukky"
             ]
         """
-        if self._series._is_polars:
-            return self._series.__class__(self._series._series.str.slice(0, n))
-        return self._series.__class__(self._series._series.str.head(n))
+        return self._series.__class__(self._series._series.str.slice(0, n))
```

Comment:
can we just call the underlying library directly here? so, keep it as it is, but in `_pandas_like`, you make `head` call `slice`

Reply:
The min version of Polars we support does not have `str.head`. If you look at the diff, it was already implemented using slice for Polars.

I thought to implement `str.slice` for pandas as well and simplify the API.

WDYT?

Reply:
I think the problem is more on the Polars side, that our min supported version does not implement `str.head`, hence we use slice.

I thought that implementing pandas `str.slice` would bring both a feature and unify the API call in here.

WDYT?

Reply:
aaah, I see, thanks, that's why we were using `.str[:n]` in the first place 😄 cool, looks fine then, thanks for explaining!